### PR TITLE
Qt5

### PR DESCRIPTION
**Pull request by endJunction**
This is an almost complete update to qt5 which is the default link library of vtk > 6.3 (and lower for some systems).

WIP:
- [ ] setup test environments
  - [ ] Travis
  - [ ] Jenkins Linux
  - [ ] Jenkins Win
  - [ ] Jenkins Mac
  - [ ] Jenkins crosscompile gcc on Win
- [ ] Installation of ogs_gui needs an update; `DeployQt4.cmake` included in `DataExplorer.cmake`
- [ ] Update documentation
- [x] cannot compile VtkVis target yet; the error was `'QWidget' file not found` when compiling `Applications/DataExplorer/VtkVis/ui_VisualizationWidgetBase.h`

### CODE REVIEW

**Comment by endJunction** [comment]
Removed the changes adding QWidgets to some includes previously. Had to add Qt5Widgets to include_directories for all of Applications/DataExplorer targets; not sure if correct place.

First time it is working for me. Thanks Lars!

File: Applications/DataExplorer/VtkVis/CMakeLists.txt [hunk 1]
```diff
@@ -116,6 +116,8 @@ include_directories(
 
 	${CMAKE_CURRENT_BINARY_DIR}/../../BaseLib
 	${CMAKE_CURRENT_BINARY_DIR}/../DataView
+
+	${Qt5Widgets_INCLUDE_DIRS}
```
Comment (bilke) [inline Applications/DataExplorer/VtkVis/CMakeLists.txt]: I think this line can be removed as it is included in the parent CMake file.

**Comment by endJunction** [comment]
Recent changes in DataExplorer require some changes to compile this after a rebase. I'm no longer able to maintain this PR. Those who are interested please make a local copy of this branch until the 1. March.

**Comment by bilke** [comment]
I copied the branch.

**Pull request closed without merging**